src/axis/axis.ts

```diff
@@ -1307,11 +1308,13 @@ export module LabelLayoutStrategy {
                 // NOTE: see note above - rotation only lines up with default d3 tickSize(6,0)
                 // TODO don't do these rotations if we already did them
                 axisLabel.style("text-anchor", "end")
-                    .attr({
-                        "dx": "-0.5em",
-                        "dy": defaultRotation.dy,
-                        "transform": defaultRotation.transform
-                    });
+                    .attr(
+                        "dx", "-0.5em")
```

Comment:
Since it's less than 120 symbol we should write this function call into a single line